capa/features/extractors/ida/extractor.py

```diff
@@ -28,10 +34,16 @@ def __init__(self):
         self.global_features.extend(capa.features.extractors.ida.file.extract_file_format())
         self.global_features.extend(capa.features.extractors.ida.global_.extract_os())
         self.global_features.extend(capa.features.extractors.ida.global_.extract_arch())
+        self.sample_hashes = SampleHashes(
+            md5=idaapi.get_input_file_md5(), sha1=idaapi.get_input_file_sha1(), sha256=idaapi.get_input_file_sha256()
```

Comment:
```suggestion
            md5=idaapi.get_input_file_md5(), sha1="(unknown)", sha256=idaapi.get_input_file_sha256()
```

i dont think IDA provides an accessor for SHA1. have you found otherwise?

Reply:
ah yeah Indeed, I think I'll revert to using the original approach instead